qube/create/templates/lib/lib_groovy/{{ cookiecutter.project_slug }}/docs/usage.rst

```diff
@@ -0,0 +1,7 @@
+=====
+Usage
+=====
+
+To use {{ cookiecutter.project_slug }} in a project::
```

Comment:
That's wrong.

That's Python legacy.

You should describe how to add it to a pom.xml or build.gradle and how to include it into the code.

Must be fixed in lib-java as well.

Reply:
see the issue linked below